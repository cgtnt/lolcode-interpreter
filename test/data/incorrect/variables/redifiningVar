HAI

I HAS A VAR ITZ 2
I HAS A VAR ITZ 3

KTHXBYE
