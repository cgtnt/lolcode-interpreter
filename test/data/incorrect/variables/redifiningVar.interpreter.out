Line 4: Cannot redefine variable VAR
